11LessonEleven/in-class-project/css/style.css

```diff
@@ -0,0 +1,61 @@
+.boxmain{
+  width:100%;
+  height:300px;
+  background-image: url(../img/sub_header_home.jpg);
+  text-align: center;
+  color:white;
+  padding-top:100px;
+}
+h1{
+  font-size: 50px;
+}
+p{
+  font-size: 14px;
+  font-family: sans-serif;
+}
+.box {
+  float:left;
+  width:25%;
+  height:150px;
+  text-align: center;
+  margin-top: 20px;
+}
+.box img{
+  padding-left:30px;
+}
+@media (max-width:768px)  and (max-width: 991px){
+    .box {
+      width:50%;
+      height:200px;
+    }
+    .boxmain {
+      background-size: cover;
+    }
+  }
+@media (max-width:767px) {
+  .box {
+    width:100%;
+    margin-top: 30px;
+  }
+  .box img{
+    padding-left:10px;
+  }
+  .boxmain {
+    background-size: cover;
+    background-position: center;
+  }
+  h1{
+    font-size:40px;
+  }
+  p{
+    font-size:10px;
+  }
+}
+.clearfix:after {
+     visibility: hidden;
+     display: block;
+     font-size: 0;
+     content: " ";
+     clear: both;
+     height: 0;
+     }
```

Comment:
literally all of my projects that I have done with bootstrap non of  the column stuff is working. It was all working before yesterday and now it is not in my live server. I have not updated anything to gitbub because i don’t want to mess up anything. So the projects for lesson 12- what we are currently working on.



> On Jun 23, 2016, at 3:17 PM, Victoria Chuang <notifications@github.com> wrote:
> 
> In 11LessonEleven/in-class-project/css/style.css <https://github.com/AustinCodingAcademy/intro-workbook/pull/6#discussion_r68306922>:
> 
> > +  }
> > +  h1{
> > +    font-size:40px;
> > +  }
> > +  p{
> > +    font-size:10px;
> > +  }
> > +}
> > +.clearfix:after {
> > +     visibility: hidden;
> > +     display: block;
> > +     font-size: 0;
> > +     content: " ";
> > +     clear: both;
> > +     height: 0;
> > +     }
> what isn't working?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/AustinCodingAcademy/intro-workbook/pull/6/files/20f1610f6cc7ffe234faacdd8cb72bc380e4ff91#r68306922>, or mute the thread <https://github.com/notifications/unsubscribe/ASRVxVO_3OLVp7ZAK82FRsZEKj7BvaJVks5qOunegaJpZM4IV2tL>.
>